inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,555 @@
+abstract: |
+  The Abstract (250 words maximum) should be structured according to the instructions for the corresponding 'papercat' (paper category, see https://academic.oup.com/gigascience/pages/instructions_to_authors).
+
+# keywords: keywords presented as a semicolon-separated list below the abstract  
+keywords:
+  - Keyword 1
+  - keyword 2
+  - keyword 3 (Three to ten keywords representing the main content of the article)
+
+# keypoints: key points presented as a bullet list in a box at the top of the second page
+keypoints:
+  - This is the first point
+  - This is the second point
+  - One last point.
+
+# highlighting-macros: list of {latex_command+}
+#   - This may be for syntax-highlighting source-code, but I'm not sure
+#   - Value is a list; each list-member is a TeX snippet
+#   - This is included just before 'header-includes'
+
+# header-includes: list of {latex_command+}
+#   - Value is a list; each list-member is a TeX snippet
+#   - You can use 'header-includes' to set or override some TeX settings, e.g.,
+#   - This is included just before 'preamble'
+#   - concerning specific keys:
+#     - journal: Only "gigascience" and "general" are implemented now.
+#     - usepackage: use packages that are not declared in the TeX template used by pandoc
+#     - bibliographystyle: apples to the natbib TeX package
+#       - see http://ctan.math.washington.edu/tex-archive/macros/latex/contrib/natbib/natbib.pdf
+header-includes:
+  - \journal{gigascience}
+  - \usepackage{natbib}
+#   # the next line is part of the document-class, oup-contemporary
+#   - \bibliographystyle{vancouver-authoryear}
+
+# preamble: Tex literal (newlines will be stripped out)  
+#   - Value is a TeX snippet that will be collapsed to a single line
+# preamble: |
+#  \setcounter{errorcontextlines}{4}
+#  \setcounter{secnumdepth}{2}
+
+# documentclass - the TeX document class - for GigaScience, must be oup-contemporary
+documentclass: oup-contemporary
+
+# paper: 'a4paper' or 'letterpaper'
+paper: letterpaper
+#paper: a4paper
+
+# reftype: 'num-refs' or 'alpha-refs', but 'alpha-refs' may not work ...
+reftype: num-refs
+#reftype: alpha-refs
+
+# In some cases, it may be easier to use TeX than rmarkdown for citations: 
+#   - TeX citations may give you more flexibility
+#   - e.g., references can be deleted or added subsequent to translation from rmarkdown to TeX
+#      - This may be important if you need to delete references on Overleaf and regenerating
+#        the TeX is not an option
+# Hence: 'bibliography' and 'csl' below are commented out because they are only needed for citations from rmarkdown
+#
+# bibliography: the file with the bibtex entries
+#bibliography: paper-refs.bib
+#
+# csl: this is needed only for citations from rmarkdown 
+#csl: gigascience.csl
+
+# output: specifies the R class used to translate rmarkdown to tex and pdf
+output: 
+  rticles::gigascience_article:
+    md_extensions: +raw_tex
```

Comment:
Actually the `raw_tex` extension is enabled by default in Pandoc, so you don't really need to enable it explicitly here.

```sh
$ echo 'Hello \LaTeX and **LaTeX**.' | pandoc -f markdown -t latex
Hello \LaTeX and \textbf{LaTeX}.
```